From e3afacf9d7ea6d7d67a7a5f53b03598264d2d9de Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 26 May 2016 10:28:57 +0800
Subject: [PATCH 1096/5242] MLK-12899-1 ARM: dts: imx7d-sdb: add dts support
 for panel 'TFT3P5581'.

commit  ad8419d3b4cb8266b69fa9e32e989745ca856f76 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the dts support for the new mipi panel 'TFT3P5581'.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-mipi-dsi.dts |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb-mipi-dsi.dts b/arch/arm/boot/dts/imx7d-sdb-mipi-dsi.dts
index 2ddf275..71d6684 100644
--- a/arch/arm/boot/dts/imx7d-sdb-mipi-dsi.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-mipi-dsi.dts
@@ -12,11 +12,15 @@
 	mipi_dsi_reset: mipi-dsi-reset {
 		compatible = "gpio-reset";
 		reset-gpios = <&gpio1 4 GPIO_ACTIVE_LOW>;
-		reset-delay-us = <50>;
+		reset-delay-us = <1000>;
 		#reset-cells = <0>;
 	};
 };
 
+&fec2 {
+	status = "disabled";
+};
+
 &lcdif {
 	disp-dev = "mipi_dsi_samsung";
 };
@@ -24,7 +28,7 @@
 &mipi_dsi {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet2_epdc0_en>;
-	lcd_panel = "TRULY-WVGA-TFT3P5079E";
+	lcd_panel = "TRULY-WVGA-TFT3P5581E";
 	resets = <&mipi_dsi_reset>;
 	status = "okay";
 };
-- 
1.7.9.5

